RecoPixelVertexing/PixelTriplets/plugins/CAHitNtupletGeneratorKernels.h

```diff
@@ -226,11 +232,11 @@ class CAHitNtupletGeneratorKernels {
 
   TupleMultiplicity const* tupleMultiplicity() const { return device_tupleMultiplicity_.get(); }
 
-  void launchKernels(HitsOnCPU const& hh, TkSoA* tuples_d, cudaStream_t cudaStream);
+  void launchKernels(HitsConstView hh, TkSoAView track_view, cudaStream_t cudaStream);
```

Comment:
I think passing the views by value is OK, but they could also be passed by (const) reference, no ?

Reply:
This was one of the things that we _think_ did not work, but this was one of the first things that we changed. We have to re-check now that everything else seems stable.